bootscript.sh

```diff
@@ -1,7 +1,5 @@
 cd ~/antora 
 
-git checkout HP_website_03092021
```

Comment:
@kgm78 though this branch doesn't exist, does it need to checkout another branch, or does it work properly?

Reply:
That line of code is not needed. It now runs based off the current branch you are working in.